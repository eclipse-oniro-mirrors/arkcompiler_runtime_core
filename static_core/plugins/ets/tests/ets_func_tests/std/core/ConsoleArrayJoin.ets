/**
 * Copyright (c) 2025 Huawei Device Co., Ltd.
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

function main() {

    let numInt: int = 1
    let strInt = "intStr"
    let numShort: short = 2
    let strShort = "shortStr"
    let numLong: long = 3
    let strLong = "longStr"
    let numFloat: float = 4.0f
    let strFloat = "floatStr"
    let numDouble: double = 5.0
    let strDouble = "doubleStr"
    let num = 1
    let str = "str"

    console.groupCollapsed(num, str)
    console.groupEnd()
    console.groupCollapsed(str)
    console.groupEnd()
    console.groupCollapsed('count %d')
    console.groupEnd()
    console.groupCollapsed()
    console.groupEnd()
    console.groupCollapsed(num, str, "sdsf")
    console.groupEnd()
    console.groupCollapsed(numInt, strInt)
    console.groupEnd()
    console.groupCollapsed(numShort, strShort)
    console.groupEnd()
    console.groupCollapsed(numLong, strLong)
    console.groupEnd()
    console.groupCollapsed(numFloat, strFloat)
    console.groupEnd()
    console.groupCollapsed(numDouble, strDouble)
    console.groupEnd()
}
