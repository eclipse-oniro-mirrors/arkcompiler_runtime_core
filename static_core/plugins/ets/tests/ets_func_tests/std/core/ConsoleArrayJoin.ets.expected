# Copyright (c) 2025 Huawei Device Co., Ltd.
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
1,str
str
count %d
1,str,sdsf
1,intStr
2,shortStr
3,longStr
4,floatStr
5,doubleStr
